wallet/src/account/mod.rs

```diff
@@ -664,6 +666,14 @@ impl Account {
             self.account_info.best_block_height(),
         )
     }
+
+    pub fn has_transactions(&self) -> bool {
+        !self.output_cache.txs().is_empty()
+    }
+
+    pub fn name(&self) -> Option<String> {
+        self.account_info.name().clone()
```

Comment:
Why not return a reference, and make the caller choose to clone?

Reply:
fixed